src/data/nsynth.py

```diff
@@ -17,15 +17,15 @@ def nsynth_from_tfrecord(nsynth_tfrecord_path):
 
 def nsynth_to_melspec(dataset, hparams):
     if 'instrument' in hparams:
-        dataset = preprocess.filter_transform(instrument_filter)(dataset)
+        dataset = pro.filter_transform(instrument_filter)(dataset)
 
     # Create preprocessing pipeline for the melspectograms
-    return preprocess.pipeline([
-        preprocess.extract('audio'),
-        preprocess.melspec(sr=hparams['sample_rate']),
-        preprocess.pad([[0, 0], [0, 4]], 'CONSTANT', constant_values=hparams['log_amin']),
-        preprocess.amp_to_log(amin=hparams['log_amin']),
-        preprocess.normalize(),
+    return pro.pipeline([
+        pro.extract('audio'),
+        pro.melspec(sr=hparams['sample_rate']),
+        pro.pad([[0, 0], [0, 4]], 'CONSTANT', constant_values=hparams['log_amin']),
+        pro.amp_to_log(amin=hparams['log_amin']),
+        pro.normalize(),
     ])(dataset)
 
 instruments = {
```

Comment:
This list seems fairly pointless to me, since the `instrument_str` key (or something) already exists.

Reply:
Also not related to this pull request, create issue